pkg/webhook/mutation/pod/common/oneagent/config.go

```diff
@@ -17,6 +17,8 @@ const (
 	// defaults to DefaultInstallPath if not set.
 	AnnotationInstallPath = "oneagent.dynatrace.com/install-path"
 
+	AnnotationVolumeType = "oneagent.dynatrace.com/volume-type"
```

Comment:
Nitpick: isn't there a const for the `oneagent.dynatrace.com` prefix?

Reply:
only for the `oneagent` part

will use that